docs/changehistory/NextVersion.md

```diff
@@ -28,7 +28,27 @@ Can now be expressed as follows (note no IModelConnection is required):
   });
 ```
 
-## Model map layers
+## Map Layers
+
+### MapLayersUI initialization
+
+In previous versions, MapLayersUI was intialized with an ever growing list of unnamed args. Now it is initialized with a single [MapLayersConfig]($map-layers) object with named args. The following code snippet demonstrates how to initialize MapLayersUI:
+
+Previously:
+
+```ts
+await MapLayersUI.initialize(false, undefined, ...);
+```
+
+Now:
+
+```ts
+await MapLayersUI.initialize({ ...MapLayersConfig });
```

Comment:
Neither of your two snippets of sample code look like they would compile. Please use working code illustrating how existing code can be upgraded to the new API.